webpages/settings/index.html

```diff
@@ -1,14 +1,87 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" dir="ltr">
   <head>
+    <meta charset="UTF-8" />
+
+    <!-- Preload styles-->
     <link rel="preload" href="light.css" as="style" />
     <link rel="preload" href="style.css" as="style" />
-    <meta charset="UTF-8" />
+
+    <!-- Preload font-->
+    <link rel="preload" href="../../libraries/thirdparty/Sora.css" as="style" />
+    <link
+      rel="preload"
+      href="../../libraries/thirdparty/Sora/latin-ext.400.woff2"
+      as="font"
+      type="font/woff2"
+      crossorigin
+    />
+    <link
+      rel="preload"
+      href="../../libraries/thirdparty/Sora/latin-ext.600.woff2"
+      as="font"
+      type="font/woff2"
+      crossorigin
+    />
+    <link
+      rel="preload"
+      href="../../libraries/thirdparty/Sora/latin.400.woff2"
+      as="font"
+      type="font/woff2"
+      crossorigin
+    />
+    <link
+      rel="preload"
+      href="../../libraries/thirdparty/Sora/latin.600.woff2"
+      as="font"
+      type="font/woff2"
+      crossorigin
+    />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title></title>
     <link rel="icon" href="../../images/icon.png" />
+
+    <!-- Preload index.js imports -->
+    <link rel="modulepreload" href="../../libraries/common/cs/download-blob.js"">
```

Comment:
Typo (should end with `download-blob.js" />`, not `download-blob.js"">`)

Reply:
Yeah, this has been very annoying in my atom editor, but my vscode editor doesn't care. Prettier doesn't care either.